app/assets/stylesheets/mixins/vars.sass

```diff
@@ -70,13 +72,21 @@ $icon-filters: '\e917'
     @content
 
 @mixin tablet-or-smaller
-  @media only screen and (max-width: #{$laptop-width - 1px})
+  @media only screen and (max-width: #{$tablet-width - 1px})

```

Comment:
The width variables represents the smallest value for that screen resolution. This change will not give the expected behaviour.

Same for the two new rules below.

Reply:
shud i remove this?
@mixin laptop-or-smaller
  @media only screen and (max-width: #{$laptop-width - 1px})

Reply:
You can keep that, but you will need to change it to 
```
@mixin laptop-or-smaller
    @media only screen and (max-width: #{$desktop-width - 1px})
```